halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/validate/v1/providers/dockerRegistry/DockerRegistryAccountValidator.java

```diff
@@ -55,6 +63,29 @@ public void validate(ConfigProblemSetBuilder p, DockerRegistryAccount n) {
       if (resolvedPassword.isEmpty()) {
         p.addProblem(Severity.WARNING, "The supplied password file is empty.");
       }
+    } else if (passwordCommandProvided) {
+      try {
+        ProcessBuilder pb = new ProcessBuilder("bash", "-c", passwordCommand);
+        Process process = pb.start();
+        int errCode = process.waitFor();
+        log.debug("Full command is" + pb.command());
+
+        if (errCode != 0) {
+          String err = IOUtils.toString(process.getErrorStream());
+          log.error("Password command returned a non 0 return code, stderr/stdout was:" + err);
```

Comment:
I might submit a "warning" here too -- even though this command might be run on a different machine (clouddriver) than halyard is currently running on.